addons/mail/static/src/components/chat_window_manager/chat_window_manager_tests.js

```diff
@@ -426,6 +429,10 @@ QUnit.test('chat window: basic rendering', async function (assert) {
         "General",
         "should have correct thread name in header part"
     );
+
+    assert.doesNotHaveClass(chatWindowHeader.querySelector(`:scope .o_ChatWindowHeader_name`), 'o-clickable',
```

Comment:
`:scope` is not necessary when the selector contains a single class name, the rest of this test is a bit obsolete so you don't need to copy this. Also it's better to use single quote instead of back tick when there is no template literal inside.

```suggestion
    assert.doesNotHaveClass(chatWindowHeader.querySelector('.o_ChatWindowHeader_name'), 'o-clickable',
```